packages/webapp/src/containers/CropCatalogue/useStringFilteredCrops.js

```diff
@@ -11,12 +11,11 @@ export default function useStringFilteredCrops(crops, filterString) {
       }
       return false;
     };
-    return crops.filter((crop) =>
-      check([
-        crop?.crop_common_name,
-        crop?.crop_variety_name,
-        t(`crop:${crop.crop_translation_key}`),
-      ]),
-    );
+    // These fields can be used in future.
+    // Hence, keeping it in the comments
+    // use it in the check function argument.
+    // crop?.crop_common_name,
+    // crop?.crop_variety_name,
+    return crops.filter((crop) => check([t(`crop:${crop.crop_translation_key}`)]));
```

Comment:
I'm not sure which of these values we need to include in the search. Will raise the issue in Jira.

Reply:
Yes, I tested out, if we want results to be related to a specific language then we need to use only t(`crop:${crop.crop_translation_key}` which will filter out tomato whenever we are performing a search in the Spanish language. 

Please check this link: 

https://playcode.io/895689

I have created a mockup to make things more clearer

Reply:
I have added crop_common_name as discuss over the call.